packages/docusaurus/lib/commands/start.js

```diff
@@ -97,43 +98,40 @@ module.exports = async function start(siteDir, cliOptions = {}) {
     config = applyConfigureWebpack(configureWebpack, config, false);
   });
 
-  const compiler = webpack(config);
-
-  // Run webpack serve.
-  await serve(
-    {},
-    {
-      compiler,
-      open: true,
-      devMiddleware: {
-        logLevel: 'silent',
-      },
-      hotClient: {
-        port: hotPort,
-        logLevel: 'error',
-      },
-      logLevel: 'error',
-      port,
-      host,
-      add: app => {
-        // Serve static files.
-        const staticDir = path.resolve(siteDir, 'static');
-        if (fs.existsSync(staticDir)) {
-          app.use(mount(baseUrl, serveStatic(staticDir)));
-        }
-
-        // Enable HTTP range requests.
-        app.use(range);
-
-        // Rewrite request to `/` since dev is only a SPA.
-        app.use(
-          convert(
-            history({
-              rewrites: [{from: /\.html$/, to: '/'}],
-            }),
-          ),
-        );
-      },
+  // https://webpack.js.org/configuration/dev-server
+  const devServerConfig = {
+    compress: true,
+    clientLogLevel: 'error',
+    hot: true,
+    quiet: true,
+    headers: {
+      'access-control-allow-origin': '*',
```

Comment:
This option was set to enable CQRS.

Reply:
I think you meant CORS. [CQRS](https://martinfowler.com/bliki/CQRS.html) is a different concept altogether.